lib/src/gal_platform_interface.dart

```diff
@@ -0,0 +1,48 @@
+import 'dart:typed_data';
+
+import 'package:plugin_platform_interface/plugin_platform_interface.dart';
+
+import 'gal_method_channel.dart';
+
+/// Plugin Platform Interface
+abstract class GalPlatform extends PlatformInterface {
```

Comment:
You might want to use the `base` keyword from dart 3

Reply:
WAIT